documentation/assemblies/deploying/assembly-deploy-kafka-cluster.adoc

```diff
@@ -5,21 +5,23 @@
 [id='kafka-cluster-{context}']
 = Deploying Kafka
 
-Apache Kafka is an open-source distributed publish-subscribe messaging system for fault-tolerant real-time data feeds.
+[role="_abstract"]
+To be able to manage a Kafka cluster with the Cluster Operator, you must deploy it as a `Kafka` resource.
+Strimzi provides example deployment files to do this.
+You can use these files to deploy the Topic Operator and User Operator at the same time.
 
-The procedures in this section describe the following:
+After you have deployed the Cluster Operator, use a `Kafka` resource to deploy the following components:
 
-* How to use the Cluster Operator to deploy:
-** xref:deploying-kafka-cluster-{context}[An _ephemeral_ or _persistent_ Kafka cluster]
-** The Topic Operator and User Operator by configuring the `Kafka` custom resource:
-*** xref:deploying-the-topic-operator-using-the-cluster-operator-{context}[Topic Operator]
-*** xref:deploying-the-user-operator-using-the-cluster-operator-{context}[User Operator]
-* Alternative standalone deployment procedures for the Topic Operator and User Operator:
-** xref:deploying-the-topic-operator-standalone-{context}[Deploy the standalone Topic Operator]
-** xref:deploying-the-user-operator-standalone-{context}[Deploy the standalone User Operator]
+* xref:deploying-kafka-cluster-{context}[Kafka cluster]
+* xref:deploying-the-topic-operator-using-the-cluster-operator-{context}[Topic Operator]
+* xref:deploying-the-user-operator-using-the-cluster-operator-{context}[User Operator]
 
 When installing Kafka, Strimzi also installs a ZooKeeper cluster and adds the necessary configuration to connect Kafka with ZooKeeper.
 
+If you haven't deployed a Kafka cluster as a `Kafka` resource, you can't use the Cluster Operator to manage it.
+This applies, for example, to a Kafka cluster running outside of Kubernetes.
+However, you can use the Topic Operator and User Operator with a Kafka cluster that is *not managed* by Strimzi, by xref:deploy-standalone-operators_{context}[deploying them as standalone components].
```

Comment:
I know this chapter focus on Kafka broker itself. But given this sentence ... should we also add that Connect, Mirror Makers or Bridge can be used with Kafka cluster not managed by Strimzi as well?

Reply:
Okay. I added _You can also deploy and use other Kafka components with a Kafka cluster not managed by Strimzi._